src/ex1.py

```diff
@@ -0,0 +1,7 @@
+def get_week_day (week: int) -> str:
```

Comment:
A voler essere precisi e per evitare situazioni impreviste a causa del controllo nell'if, il return type sarebbe:

```py
def get_week_day (week: int) -> str | None:

# Oppure, anche se ormai un po' vecchiotto

from typing import Optional

def get_week_day (week: int) -> Optional[str]
```

(PHPStan is always watching...)

Reply:
Bon'anima